scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -47,6 +54,13 @@ namespace <%= namespace %> {
         public Cart(ShopifyClient client) {
             Client = client;
             _LineItems = new CartLineItems();
+
+            // TODO: When we add more platform we can move this into a factory.
+            #if UNITY_IOS
+            checkout = new iOSCartCheckout();
```

Comment:
These `Checkout` classes are going to need a reference to `Cart`, so it can make mutations on the cart. 
Or they need to set the `Cart` as a delegate to checkout events